static/themes/SSF/style.css

```diff
@@ -46,3 +46,9 @@
     background-image: url(../../img/sunflower_spin_throbber.gif);
     background-repeat: no-repeat;
 }
+
+/* Comments */
+
+#comment-form {
+    width: 100%;
```

Comment:
@hitesh96db I don't think I will be able to do that... Can you give me something of difficulty similar to my previous ticket "My Task"?